engine/src/main/java/org/pentaho/di/job/JobMeta.java

```diff
@@ -2181,7 +2182,7 @@ public List<StringSearchResult> getStringList( boolean searchSteps, boolean sear
         // if ( includePasswords )
         // {
         if ( meta.getPassword() != null ) {
-          stringList.add( new StringSearchResult( meta.getPassword(), meta, this,
+          stringList.add( new StringSearchResult( Strings.repeat("*", meta.getPassword().length()), meta, this,
```

Comment:
Change is fine, please update the copyright header in this file.

Reply:
updated copyright header